lib/Pages/Profil/profil.dart

```diff
@@ -40,7 +40,7 @@ class _ProfileState extends State<Profile> {
   void initState() {
     super.initState();
     this.userInfo = widget.moreaFire.getUserMap;
-    this._getNachrichtenGruppen();
+    //this._getNachrichtenGruppen();
```

Comment:
Das funktioniert no ohni dass du NachrichtenGruppen setzisch?
Wil das wird i minere Version benutzt zum d Lischte erstelle, wo azeigt wird im ListTile vo Nachrichtengruppen.

Reply:
das holt ja d messageGroups us em userMap. Ich han das uskommentiert, will ich in erinnerig gha han, dass mir das nümme über messageGroups mached.

Reply:
ja aber denn muess mer das umprogrammiere oder ned.
Wil so laufts ja denn nöd, es bechunnt ja denn kein iitrag für nachrichtenGruppen.
Oder hesch du das apasst und ich gsehns ned?

Reply:
Nei muss ahpasst werde